pages/templates/pages/microsite_page.html

```diff
@@ -1,12 +1,210 @@
 {% extends "pages/base_page.html" %}
 
-{% load wagtailcore_tags %}
+{% load wagtailcore_tags wagtailimages_tags %}
+
+{% block meta %}
+  {{ block.super }}
+  <style>
+    #custom_header {
+      {% if page.custom_header_background_color %}
```

Comment:
Instead of if else, could use `firstof` pattern?

Reply:
Two reasons I moved away from `firstof` here:
1. It was borking syntax highlighting through entire document for some reason, which was making code editing extremely confusing. I guess because it was in HTML?
2. I didn't want to hard code any values. So for example `firstof text_color FFFFF` would resort to white if no text_color was present, but really we want it to result to whatever color is in the global styles.